utilities/validations.py

```diff
@@ -125,3 +125,17 @@ def validate_room_labels(**kwargs):
         if structure is None:
             raise GraphQLError("Structure does not exist")
         break
+
+
+def validate_structure_id(**kwargs):
+    """
+    Function to validate that a structure id exists in structures
```

Comment:
Kindly change the doc string to indicate that validation is being done against the `office_structure` table. I'm aware there's a task to remove the ambiguity that exists between the `structure` and `office_structure` models, so in the meantime just update the doc string alone you can leave the variable names.
Otherwise, nice work 👍